R/get_xronos.R

```diff
@@ -0,0 +1,36 @@
+#' @rdname db_getter_backend
+#' @export
+get_xronos <- function(db_url = get_db_url("xronos")) {
+
+  check_connection_to_url(db_url)
+
+  resp<-httr::GET(db_url)
```

Comment:
Here comes a big one: I know you like proper APIs - rightfully so - but for c14bazAAR they are not ideal. Is there a chance we could get a setup like the one for radon again? Every night the server creates a simple .csv flat file with the columns we need?

Maybe I'm completely off and this is already the fastest solution. Happy to be disabused. But I have the feeling the simple interface would be significantly faster, even for the massive amount of data in xronos. Maybe c14bazAAR is too specific of an application to consider this.

Reply:
I know it's unfair, but look at that:

```
> system.time(get_calpal() -> test1)
   user  system elapsed 
  0.430   0.027   0.417 
> system.time(get_radon() -> test3)
   user  system elapsed 
  0.444   0.045   0.887 
> system.time(get_xronos() -> test2)
   user  system elapsed 
 75.529   0.617 188.656 
```
This version of CalPal has 49834, so more than half of the 92126 dates in xronos.